tempo_embeddings/text/abstractcorpus.py

```diff
@@ -191,7 +200,7 @@ def set_topic_label(self, **kwargs) -> None:
             stopwords: if given, exclude these words
         """
 
-        self._label = "; ".join(self.top_words(**kwargs))
+        self._label = "; ".join(sorted(self.top_words(**kwargs)))
```

Comment:
Does this imply alphabetical sorting? I don't quite understand why.

Reply:
Ah, if this is about testing, I think it is better to sort on the test side, rather than changing the actual code for the sake of testing.

Reply:
actually, I changed it for stats and viz purposes, I think it is more consistent if the cluster names are always sorted, that way it is easier to find them for example when looking at the Bookeh figure and looking at the CSV file with the cluster contents

Reply:
Aren't they sorted by their c-TFIDF values already? If not, perhaps that should be fixed. Perhaps some mechanism for tie-breaking needs to be added, or I am missing something else.
Anyway, I understand that this might be out of scope for now.